docs/howto/operate/new-cluster/aws.md

```diff
@@ -128,6 +134,13 @@ role - and users can modify them as they wish.
 
 ### Create account with finely scoped permissions for automatic deployment
 
+<!-- Running terraform for AWS code now does more than *just* create a CI/CD account -
```

Comment:
I think?

Reply:
It does:

1. Create CI/CD role 
2. Setup EFS in appropriate networked place
3. Optionally set up a shared database
4. Optionally set up user buckets

I think these are all documented but scattered

Reply:
Ok, I will track these docs down and link them. And restructure this section a little bit to make more sense.